backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5681990_cli_gh14915_fix_missing_column_in_ad_field_v.sql

```diff
@@ -0,0 +1,188 @@
+drop view if exists ad_field_v;
```

Comment:
@teosarca they're already up to date

Reply:
ok... but then what's the purpose of that script? isn't the DB version in line with what we have in ddl folder?

(btw, it shall be `_sys_` instead of `_cli_`)

Reply:
the latest migration script didn't have the first column, i documented it here https://github.com/metasfresh/me03/issues/14833#issuecomment-1473744584